types/index.d.ts

```diff
@@ -198,7 +198,7 @@ declare namespace nock {
     once(): this
     twice(): this
     thrice(): this
-    optionally(): this
+    optionally(flag?: boolean): this
```

Comment:
confirmed, it's documented as such

<img width="585" alt="image" src="https://user-images.githubusercontent.com/39992/130155608-2ddb6125-fc5c-442e-af1e-e59f3ca9863e.png">